client/Panels/Home/Home.tsx

```diff
@@ -3,14 +3,22 @@
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
 import React, { FunctionComponent } from 'react';
+import get from 'lodash.get';
 import image from 'Images/logo.png';
 import './style.scss';
+import { useConfigFeatureFlag } from 'Hooks';
 
 const Home: FunctionComponent = ({ children }) => {
+  const { client, featureFlags, isComplete } = useConfigFeatureFlag();
```

Comment:
Is the design that any component wanting to use flags will need loading indicators?